sklearn/utils/extmath.py

```diff
@@ -600,6 +600,7 @@ def log_logistic(X, out=None):
     """
     is_1d = X.ndim == 1
     X = check_array(X, dtype=np.float)
+    X = np.atleast_2d(X)
```

Comment:
I don't understand. If ``is_1d`` it is squeezed below, right?
Where is this called with a 1d X and why is that 1d?